src/main/java/com/comphenix/protocol/utility/MinecraftReflection.java

```diff
@@ -176,28 +171,7 @@ public static String getMinecraftPackage() {
 				Method getHandle = getCraftEntityClass().getMethod("getHandle");
 				MINECRAFT_FULL_PACKAGE = getHandle.getReturnType().getPackage().getName();
 
-				// Pretty important invariant
-				if (!MINECRAFT_FULL_PACKAGE.startsWith(MINECRAFT_PREFIX_PACKAGE)) {
```

Comment:
this was added somewhat recently to fix issues with forge servers. merge conflict, or does this cause an issue in 1.19.3?